docs/guides/burst_insar_product_guide.md

```diff
@@ -527,7 +527,16 @@ files.
 | Radar sensing stop              | Last date and time for data collection                                                                  | 2020-06-04T02:23:16.030988                                           |
 
 *Table 6: List of additional InSAR parameters included in the parameter text file `INSAR_ISCE_BURST` job types.*
+{% endblock %}
+
+{% block download_tutorial %}
+Step-by-step instructions for finding and downloading RTC On Demand products in Vertex are available in the 
+[Downloading On Demand Products](https://storymaps.arcgis.com/stories/191bf1b6962c402086807390b3ce63b0#ref-n-4rF6zW "Burst-Based InSAR for Sentinel-1 On Demand StoryMap Tutorial - Downloading On Demand Products"){target=_blank} 
+section of the 
+[Burst-Based InSAR for Sentinel-1 On Demand](https://storymaps.arcgis.com/stories/191bf1b6962c402086807390b3ce63b0 "Burst-Based InSAR for Sentinel-1 On Demand StoryMap Tutorial" ){target=_blank} interactive StoryMap Tutorial.
```

Comment:
should the T in tutorial be capitalized?
```suggestion
[Burst-Based InSAR for Sentinel-1 On Demand](https://storymaps.arcgis.com/stories/191bf1b6962c402086807390b3ce63b0 "Burst-Based InSAR for Sentinel-1 On Demand StoryMap Tutorial" ){target=_blank} interactive StoryMap tutorial.
```

Reply:
addressed in [this commit](https://github.com/ASFHyP3/hyp3-docs/pull/590/commits/c872114b4e9ae6f0d89bc8deb063e8917e0d9ac2)